bears/vcs/git/GitCommitBear.py

```diff
@@ -260,6 +268,25 @@ def check_body(self, body,
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
 
+        if valid_email_check:
+            result_message = 'Body contains these invalid emails:\n'
+            invalid_emails = False
+            for line in body:
+                for email in re.findall(r'\S*@\S*', line):
+                    email_check = re.match(r"(^[a-zA-Z0-9_.+-]"
+                                           +r"+@[a-zA-Z0-9-]+\."
```

Comment:
You do not use the preferred quotation marks.

*QuotesBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/bears/vcs/git/GitCommitBear.py
+++ b/bears/vcs/git/GitCommitBear.py
@@ -274,7 +274,7 @@
             for line in body:
                 for email in re.findall(r'\S*@\S*', line):
                     email_check = re.match(r"(^[a-zA-Z0-9_.+-]"
-                                           +r"+@[a-zA-Z0-9-]+\."
+                                           +r'+@[a-zA-Z0-9-]+\.'
                                            +r"[a-zA-Z0-9-.]+$)", email)
                     # if eamil do not match regex there is no need to
                     # look up dns
```